articles/what-is-jenkins/index.md

```diff
@@ -0,0 +1,61 @@
+    alt: Jenkins Continuous Integration tool
+---
+Jenkins is a Continuous Integration (CI) tool and an open-source tool written in Java. Jenkins achieves Continuous Integration with the help of plugins, which are over a thousand in count. Developers mostly use Jenkins for Continuous Integration because of its flexibility and the number of plugins it supports.
+<!--more-->
+### What is Jenkins?
+Developers use Jenkins to continuously build and test software projects, making it easier to integrate changes to the project, and making it easier for users to obtain a good build with no errors and bugs. Jenkins allows developers to continuously deliver a software project by integrating with a large number of development, deployment, and testing technologies such as Git, Maven, Puppet, Selenium, Ansible, and Nagios.
+
+### Problems Developers faced before Continuous Integration
+Earlier before Continuous Integration was introduced, a team of developers used to work from different locations, each of them would individually implement a few classes. They would write code and tests to ensure that their classes work properly. Then assume the software project will build and run smoothly because classes are tested well. They then integrate the classes and combine them to create a larger program that was being developed by the team, but suddenly everything breaks.  There are issues with integration, the code fails to compile, all sorts of bugs arise, and things stop working out. This is also called [integration hell](https://www.solutionsiq.com/agile-glossary/integration-hell/#:~:text=Integration%20Hell%20refers%20to%20the,that%20it%20can%20finally%20integrate.)
+Integration hell is a major problem to software projects and it is risky to their success reason. Seeing it was hard to estimate how long it would take to fix the things that went wrong. Developers had to figure out what went wrong, and that could be very hard to do in a large project. After diagnosing the problems, again time had to be spent fixing them, the project budget may also increase and the success of the project is risked. The solution to integration hell was to integrate continuously throughout the entire project, and thus the practice of Continuous Integration was introduced to the DevOps field.
+
+### What is Continuous Integration?
+CI is a development practice whereby developers are required to commit changes to the source code in a shared repository more frequently. CI says developers must integrate code early and often, so every time changes are pushed to the repository (such as GitHub) the project code should be checked into, compiled, and tested. The productivity of developers has been improved by Continuous Integration, rather than before when they used to wait until all components are developed. Then spend weeks and months working on bugs and errors that would [inevitably arise](https://www.edureka.co/blog/what-is-jenkins/).
```

Comment:
The ordering of how you present information is confusing. This should be one of the first paragraphs after the Jenkins intro to give readers context. An additional detail or statistic about just how impactful CI has been and how much time or productivity gains developers have seen would really highlight its utility. Also be sure to use complete sentences and not phrases - see last sentence.